docs/reference/ilm/apis/explain.asciidoc

```diff
@@ -111,30 +111,33 @@ that the index is managed and in the `new` phase:
   "indices": {
     "my-index-000001": {
       "index": "my-index-000001",
-      "managed": true, <1>
-      "policy": "my_policy", <2>
-      "lifecycle_date_millis": 1538475653281, <3>
-      "age": "15s", <4>
+      "index_creation_date_millis": 1538475653281,
+      "index_age": "15s",                       <1>
```

Comment:
@jrodewig @dakrone what do you think about the naming of these fields? We'll document they will be used by rollover in the age math for conditions, whilst the existing `age` for phase transition timings.

Do you think this is clear enough?

Reply:
Tagging @debadair, who has more ILM experience and can probably give better feedback. :)

Instead of `index_age`, I'd go with something a little more descriptive like `time_since_index_creation` or `time_since_creation`. It's more descriptive and avoids conflation with `age` and `min_age`.

Ideally, we would change `age` to a more descriptive key too, but that would require a breaking change.

Reply:
I agree that I think `index_age` is a little confusing, perhaps something like `creation_age`, or `lifetime` would be my suggestions.

Reply:
I do like that name, it does feel a bit weird that it is not inline with the other `age` though. Is there any convention we use for such fields in our APIs?

Reply:
I think I could justify a breaking change to the `age` key name assuming 
1) there are not any known consumers of this API (other then humans )
2) it can help make a meaningful difference how we express timings. 

For example, if makes sense to keep index_age here but change `age` to `phase_age` in 8.0.0 I think that should be on the table too.

Reply:
Thank you for the ideas, I will bring these options to the team and come back with a proposal.

Reply:
Our proposal is:
- `age` remains the same because it's already used by kibana so breaking backwards compatibility is not as painless.
- `time_since_index_creation` will be the name of the new exposed field to make it as clear as possible and we intentionally want to avoid the word age so the two will not be easier distinguishable.

Thank you all for helping out :).